rep-0105.rst

```diff
@@ -54,6 +54,9 @@ The ``map`` frame is useful as a long-term global reference, but
 discrete jumps make it a poor reference frame for local sensing and
 acting.
 
+It is recommended to align the map with east, north, up (ENU) as a default for consisent viewing, but is not required.
```

Comment:
What exactly do you mean by "align"? Is it enough that the x and y axes are aligned to east, north, south, west (4 possible orientations) or more specific x=East, y=North and z=Up? Some readers might interpret this as x=North because of the default camera perspective in rviz or other map visualization tools.

More important, is there also a clear definition or recommendation for the z level of the `map` frame relative to the earth or is this purely defined by the earth to map transform? The current convention seems to be that the origin of the map is at the initial level of the robot that created the map or some other local reference, so the static transform publisher would have to know the altitude of the map above the ellipsoid. See also my [related comment in Discourse](http://discourse.ros.org/t/proposed-standard-interface-for-aerial-vehicles-in-ros/287).